### ISSUE TITLE
Magento 2.2.x

### ISSUE THREAD

**Issue by adamlavery** [Worldpay/Worldpay-Magento2-CG#5]
Is the module compatible with M2.2? I appreciate you say M2.1.8+, but there's been some significant changes in M2.2 that has been causing some issues with previously M2.1 compatible extensions.

**Comment by lochansim** [comment]
Hi Adam, 
We've just entered UAT for the work to upgrade from 2.1.x. We're expecting to release the plugin, and documentation, on github w.c.19th February.
Let me know if you have any questions

**Comment by adamlavery** [comment]
Hi, can you confirm the new version supports PHP 7.1?

**Comment by lochansim** [comment]
I can confirm yes. We are verifying this as part of our standard UAT process

**Comment by adamlavery** [comment]
Any update on this? Tks.

**Comment by RGV500** [comment]
I too am waiting for this release to enable me to test Worldpay on my M2.2 website

Any further news on its release please ??

**Comment by settoloki** [comment]
I'm also waiting for a release to support 2.2.3 and php 7.1, it's stopping a site going live, do we have any more news on this?

**Comment by adamlavery** [comment]
Can we get an update on this please? Now well beyond expected delivery date and clients need to know what's happening in case we need to consider alternatives to WorldPay.

**Comment by settoloki** [comment]
if you change

`$siteCodes = unserialize($sitecodeConfig);`

to

```
$json = new Magento\Framework\Serialize\Serializer\Json;
$siteCodes = $json->unserialize($sitecodeConfig);

```
on line 80 of **worldpay > magento2-module-payments > Model > Config.php**

it will run on php 7.1/7.2 and not error, it's not idea but it works. I have found no issues "so far" on the site I had to make this change on.

I'm the first person to kick off if someone changes a vendor file though.
